app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -22,8 +22,9 @@
   error_id = "error-#{SecureRandom.hex(4)}"
   search_icon ||= nil
   heading_size = false unless ['s', 'm', 'l', 'xl'].include?(heading_size)
+  prefix ||= nil
   suffix ||= nil
-
+  
```

Comment:
Stray whitespace

Reply:
I think this is okay, there was already a space between 'suffix' and 'css_classes', I assume for readability.

Reply:
I meant the tab character, the newline is fine.

Reply:
🤦 oh i'm an idiot. Thanks will change now